bower.json

```diff
@@ -4,9 +4,10 @@
   "authors": [
     "Konstantin Zudov <co@zudov.me"
   ],
+  "keywords": ["pursuit-deprecated"],
   "repository": {
     "type": "git",
-    "url": "git://github.com/zudov/purescript-strongcheck-generics.git"
+    "url": "https://github.com/zudov/purescript-strongcheck-generics.git"
```

Comment:
The repository URL was updated to match the one in the [registry](https://github.com/purescript/registry/blob/master/bower-packages.json#L1245) so that a new version of the package can be published.